index.dd

```diff
@@ -100,6 +96,148 @@ void main()
 
 )
 )))
+$(SCRIPT
+// Picking the frontpage example after the page has loaded leads to reflows
+// and rerendering. Hence, we pick the example as soon as possible
+/**
+Bootstrap a parent node containing a <pre> element into an editor instance.
+codeBlock = parent node (usually of class `runnable-examples`)
+*/
+function buildRunnableExample(codeBlock) {
+    // helper method to construct a button in vanilla JavaScript
+    function createButton(className, value) {
+        var btn = document.createElement("input");
+        btn.type = "button";
+        btn.className = className;
+        btn.value = value;
+        return btn;
+    }
+    // helper method to construct a textarea in vanilla JavaScript
+    function createTextArea(className, value, text) {
+        var el = document.createElement("textarea");
+        el.className = className;
+        el.innerText = text;
+        return el;
+    }
+    // helper method to construct an editor overlay in vanilla JavaScript
+    // typically overlays are output,args,stdin and the editor instance
```

Comment:
Everything that "lays over" the code box, e.g.

![image](https://user-images.githubusercontent.com/4370550/27408961-53cc72b2-56df-11e7-8e13-aa4c4f176612.png)
![image](https://user-images.githubusercontent.com/4370550/27408964-5a4bcb38-56df-11e7-8e02-5aff64d84b1b.png)
![image](https://user-images.githubusercontent.com/4370550/27408976-70a1a024-56df-11e7-9a87-c1ca9241d1d3.png)

As mentioned, there weren't any comments, so I have chosen my own terminology. Better ideas?